documentation/manual/source/pages/development/icon_fonts.rst

```diff
@@ -24,7 +31,9 @@ address all the glyphs by their name/alias.
 
 The itegration is generic, so that it does not collide with the frameworks :doc:`appearance themes
 </pages/desktop/ui_theming>`. Font icons can be addressed using ``@FontName/GlyphName`` or
-``@FontName/HexUnicode`` in the source property of your ``qx.ui.basic.Image``.
+``@FontName/HexUnicode`` in the source property of your ``qx.ui.basic.Image``. To override the
+default size, you can also use ``@FontName/GlyphName/size`` or ``@FontName/HexUnicode/size`` to scale it
+to the given amount of pixels.
```

Comment:
given `number` of pixels. 

('number' is used when you can count them, 'amount' when not really countable, e.g., "the `number` of bottles in a dozen is 12" but "one cup of sugar is that `amount`"... although if you counting the grains of sugar, you'd say "the `number` of grains of sugar in one deciliter is 897,000")

Reply:
Fixed these. Looks like english courses are required ;-)